service/src/main/java/uk/gov/hmcts/reform/fpl/enums/CaseRole.java

```diff
@@ -6,6 +6,16 @@
     EPSMANAGING,
     LAMANAGING,
     SOLICITOR,
+    // Disabling enum naming convention check (Java S115) as ALL CAPS makes this unreadable
+    solicitorA, //NOSONAR
+    solicitorB, //NOSONAR
+    solicitorC, //NOSONAR
+    solicitorD, //NOSONAR
+    solicitorE, //NOSONAR
+    solicitorF, //NOSONAR
+    solicitorG, //NOSONAR
+    solicitorH, //NOSONAR
+    solicitorI, //NOSONAR
```

Comment:
this looks a bit weird, for now we dont store enum directly in case data, but use 

public String formattedName() {
        return formattedName;
    }
    public static CaseRole from(String name) {
        return CaseRole.valueOf(name.replaceAll("[\\[\\]]", ""));
    }
    
    so maybe these names can be normalised, and 2 methods can be adopted to produce correct result